src/neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -101,7 +101,7 @@ public static ContractManifest FromJson(JObject json)
                 Abi = ContractAbi.FromJson(json["abi"]),
                 Permissions = ((JArray)json["permissions"]).Select(u => ContractPermission.FromJson(u)).ToArray(),
                 Trusts = WildcardContainer<UInt160>.FromJson(json["trusts"], u => UInt160.Parse(u.AsString())),
-                Extra = json["extra"]
+                Extra = json["extra"]?.ToString() ?? "null"
```

Comment:
It will still accept `"extra":{"a": "b"}`, just parse/stringify it, but we need to have proper `JString` in `json["extra"]` to avoid this problem.